src/components/radio/RadioButton.tsx

```diff
@@ -31,6 +31,7 @@ const RadioButton = ({
         {/* eslint-disable-next-line jsx-a11y/label-has-for */}
         <label className="radio-label">
             <input
+                aria-label={name}
```

Comment:
Does the `label` element here not sufficiently handle the child `input`? Do we just need to add a `for` or something to connect them?

Reply:
@laughnan I just added `htmlfor` to connect the input and label as you recommended.